### PR TITLE
Update AWS private operator docs

### PR DESCRIPTION
**Pull request by alex-yau-ttd**
- Update healthcheck section
- Reflect changes made to CloudFormation templates

### CODE REVIEW

File: docs/guides/operator-guide-aws-marketplace.md [hunk 1]
```diff
@@ -110,24 +110,21 @@ To subscribe and deploy one or more UID2 Operators on AWS, complete the followin
...
+The following table lists all resources that are created during the [deployment](#deployment).
+
+| Name | Type | Description |
+|:------|:------|:-------------|
+| `KMSKey` | `AWS::KMS::Key` | The key for secret encryption (for configuration strings). |
+| `SSMKeyAlias` | `AWS::KMS::Alias` | An alias that provides an easy way to access the [KMS](https://aws.amazon.com/kms/) key. |
+| `TokenSecret` | `AWS::SecretsManager::Secret` | An encrypted configuration that includes the operator key. |
+| `WorkerRole` | `AWS::IAM::Role` | The IAM role that your UID2 Operators run as. Roles provide access to configuration keys. |
+| `WorkerInstanceProfile` | `AWS::IAM::InstanceProfile` | The instance profile with Worker Role to attach to Operator EC2 instances. |
+| `VPC` | `AWS::EC2::VPC` | Virtual Private Cloud (VPC) is a virtual private network that hosts private operators. You can customize and use an existing VPC as well. See also [VPC Chart](#vpc-chart).|
```
Comment (abuabraham-ttd) [inline docs/guides/operator-guide-aws-marketplace.md:122]: Not created

File: docs/guides/operator-guide-aws-marketplace.md [hunk 2]
```diff
@@ -110,24 +110,21 @@ To subscribe and deploy one or more UID2 Operators on AWS, complete the followin
...
 | `InternetGateway` | `AWS::EC2::InternetGateway` | The Internet Gateway that allows operators to communicate with the UID2 Core Service and download security updates. | Conditionally|
-| `AttachGateway` | `AWS::EC2::VPCGatewayAttachment` | A value that associates the Internet Gateway with the VPC. | Conditionally |
-| `SecurityGroup` | `AWS::EC2::SecurityGroup` | A security group policy that provides rules for operator instances. See also [Security Group Policy](#security-group-policy).| Always |
-| `LaunchTemplate` | `AWS::EC2::LaunchTemplate` | A launch template with all configurations in place. You can spawn new UID2 Operator instances from it. | Always |
-| `AutoScalingGroup` | `AWS::AutoScaling::AutoScalingGroup` | An auto-scaling group (ASG) to which the launch template is attached. You can use this to update the desired number of instances later, if needed. | Always |
+| `AttachGateway` | `AWS::EC2::VPCGatewayAttachment` | A value that associates the Internet Gateway with the VPC. |
```
Comment (abuabraham-ttd) [inline docs/guides/operator-guide-aws-marketplace.md:124]: Internet gateway is also not created

File: docs/guides/operator-guide-aws-marketplace.md (hunk 1 above)
Comment (alex-yau-ttd) [reply docs/guides/operator-guide-aws-marketplace.md]: removed VPC

File: docs/guides/operator-guide-aws-marketplace.md (hunk 2 above)
Comment (alex-yau-ttd) [reply docs/guides/operator-guide-aws-marketplace.md:122]: removed InternetGateway. Is AttachGateway still valid?

File: docs/guides/operator-guide-aws-marketplace.md [hunk 3]
```diff
@@ -208,44 +203,38 @@ The following table explains the parameter values that you need to provide in st
...
+To check the UID2 Operator status of your Load Balancer, complete the following steps:
 
-1. On the Specify group details page, select **Instances** as the target type, then enter `UID2ALBTG` as the **Target group name** and select **HTTP1** as the **Protocol version**.
-2. Under **Health checks**, provide `/ops/healthcheck` as the **Health check path**, and then expand the **Advanced health check settings** section. 
-3. Select **Override** as the **Port** and change the default value to `9080`.
-4. Select UID2 Operator EC2 Instances created by your auto-scaling group and then click **Include as pending below**. 
-5. Make sure the **Ports for the selected instances** contains `80`.
-6. Click **Create target group**.
+1. Identify the DNS name of your load balancer by going to **EC2 > Load balancers** and looking at the **DNS name** column of your load balancer.
+2. In your browser, go to `http://{dns-name-of-your-load-balancer}/ops/healthcheck`. A response of `OK` indicates good operator status.
```
Comment (abuabraham-ttd) [inline docs/guides/operator-guide-aws-marketplace.md:238]: make this https:

File: docs/guides/operator-guide-aws-marketplace.md (hunk 2 above)
Comment (abuabraham-ttd) [reply docs/guides/operator-guide-aws-marketplace.md:122]: remove AttachGateway

Comment (alex-yau-ttd) [reply docs/guides/operator-guide-aws-marketplace.md]: done

File: docs/guides/operator-guide-aws-marketplace.md (hunk 3 above)
Comment (alex-yau-ttd) [reply docs/guides/operator-guide-aws-marketplace.md:237]: done

**Review by genwhittTTD — APPROVED** [review]

**Pull request merged**